pkg/server/options/authorization.go

```diff
@@ -173,6 +173,7 @@ func (s *Authorization) ApplyTo(ctx context.Context, config *genericapiserver.Co
 		if err != nil {
 			return err
 		}
+		authorizer = authz.WithWarrantsAndScopes(authorizer)
```

Comment:
For my understanding here: We are only wrapping the (optional) webhook authorizer because the RBAC-based ones (the whole `chain` thing) got warrant support by amending the rule resolver in the kube fork, correct?

Reply:
Correct. And we don't want that external, third party authorizers have to understand scopes and warrants.